ramp-frontend/ramp_frontend/utils.py

```diff
@@ -67,6 +68,6 @@ def send_mail(to, subject, body):
         msg = Message(subject)
         msg.body = body
         msg.add_recipient(to)
-        mail.send(msg)
+        current_app.pool.submit(copy_current_request_context(mail.send), msg)
     except Exception as e:
         logger.error("Mailing error: {}".format(e))
```

Comment:
Good practice to make sure we have both logs and we are reliable in sending email/not crashing the server:

- we should add a `ensure_running` function that restart the pool if it is not running.
- we should add a way to log exceptions in send mail, maybe simply by putting the try except in the threadpool

```suggestion
        def send(msg):
            try:
                mail.send(msg)
            except Exception as e:
                logger.error("Mailing error: {}".format(e))

    current_app.pool.submit(copy_current_request_context(mail.send), msg)
```